### PR TITLE
[Improve] generate and return random password when reset user password

### PR DESCRIPTION
**Pull request by zhoulii**
## What changes were proposed in this pull request

generate and return random password to solve possible security issue.

en

Zh

## Verifying this change

- *Manually verified the change by testing locally.*

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (**yes** / no)

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/entity/User.java [hunk 1]
```diff
@@ -49,6 +49,8 @@ public class User implements Serializable {
 
   public static final String SEX_UNKNOWN = "2";
 
+  public static final Integer DEFAULT_PASSWORD_LENGTH = 8;
+
   public static final String DEFAULT_PASSWORD = "streampark666";
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/entity/User.java]: delete line 54.

File: streampark-console/streampark-console-webapp/src/views/system/user/User.vue [hunk 2]
```diff
@@ -160,8 +160,14 @@
...
         const hide = createMessage.loading('reseting');
         try {
-          await resetPassword({ usernames: record.username });
-          Swal.fire(t('system.user.table.resetSuccess', [record.username]), '', 'success');
+          const resp = await resetPassword({ username: record.username });
+          if (resp.data.code == 200) {
+            Swal.fire({
+              icon: 'success',
+              title: 'New Password',
+              html: '<pre class="api-exception"> The new password is: ' + resp.data.data + '</pre>',
```
Comment (wolfboys) [inline streampark-console/streampark-console-webapp/src/views/system/user/User.vue]: need to support i18n.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/entity/User.java (hunk 1 above)
Comment (zhoulii) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/entity/User.java]: done

File: streampark-console/streampark-console-webapp/src/views/system/user/User.vue (hunk 2 above)
Comment (zhoulii) [reply streampark-console/streampark-console-webapp/src/views/system/user/User.vue]: done

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**